### PR TITLE
Added "Waze travel time" sensor documentation

### PR DESCRIPTION
**Pull request by Myrddyn1**
**Description:**
Documentation for Waze Travel Time sensor.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant):** home-assistant/home-assistant#11870

### CODE REVIEW

File: .rake_tasks~ [hunk 1]
```diff
@@ -0,0 +1,19 @@
+clean
```
Comment (frenck) [inline .rake_tasks~]: This file should not be here?

File: source/_components/sensor.waze_travel_time.markdown [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+description: "Instructions on how to add Waze travel time to Home Assistant."
+date: 2018-01-23 00:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: waze.png
+ha_category: Transport
+ha_iot_class: "Cloud Polling"
+ha_release: 0.61
```
Comment (frenck) [inline source/_components/sensor.waze_travel_time.markdown]: `0.61` is already shipped some time ago. Please set it to `0.63`

File: source/_components/sensor.waze_travel_time.markdown [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    destination: Québec, QC
+    region: 'US'
+    update_interval: '00:03'
+    outputs:
+     - duration
+     - distance
+     - route
+```
+
+Configuration variables:
```
Comment (frenck) [inline source/_components/sensor.waze_travel_time.markdown]: Please use the configuration tags, for more information please see:
https://home-assistant.io/developers/documentation/create_page/#configuration

File: source/_components/sensor.waze_travel_time.markdown [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+- **origin** (*Required*): Enter the starting address or the GPS coordinates of the location (GPS coordinates has to be separated by a comma).
+- **destination** (*Required*): Enter the destination address or the GPS coordinates of the location (GPS coordinates has to be separated by a comma).
+- **region** (*Required*): Choose one of the available regions from 'EU', 'US', 'NA' (equivalent to 'US'), 'IL'. Default value is 'US'.
+- **outputs** (*Required*): At least one output type (see below) is required.
+  - *duration*: The sensor will display the duration of the best chosen route, in min.
```
Comment (frenck) [inline source/_components/sensor.waze_travel_time.markdown]: - `best chosen` -> `best-chosen`

File: source/_components/sensor.waze_travel_time.markdown [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+  - *distance*: The sensor will display the distance of the best chosen route, in km.
```
Comment (frenck) [inline source/_components/sensor.waze_travel_time.markdown]: - `best chosen` -> `best-chosen`

File: source/_components/sensor.waze_travel_time.markdown [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+  - *route*: The sensor will display the main steps of the best chosen route.
```
Comment (frenck) [inline source/_components/sensor.waze_travel_time.markdown]: - `best chosen` -> `best-chosen`

**Review by frenck — CHANGES_REQUESTED** [review]
Thank you for the PR @Myrddyn1.
I've reviewed it and left some comments, could you please take a look? 👍

File: .rake_tasks~ (hunk 1 above)
Comment (Myrddyn1) [reply .rake_tasks~]: I deleted the file

File: source/_components/sensor.waze_travel_time.markdown (hunk 2 above)
Comment (Myrddyn1) [reply source/_components/sensor.waze_travel_time.markdown]: Done

File: source/_components/sensor.waze_travel_time.markdown (hunk 3 above)
Comment (Myrddyn1) [reply source/_components/sensor.waze_travel_time.markdown]: I've made the requested changes using the new tags

File: source/_components/sensor.waze_travel_time.markdown (hunk 4 above)
Comment (Myrddyn1) [reply source/_components/sensor.waze_travel_time.markdown]: Corrected

File: source/_components/sensor.waze_travel_time.markdown (hunk 5 above)
Comment (Myrddyn1) [reply source/_components/sensor.waze_travel_time.markdown]: Corrected

File: source/_components/sensor.waze_travel_time.markdown (hunk 6 above)
Comment (Myrddyn1) [reply source/_components/sensor.waze_travel_time.markdown]: Corrected

File: source/_components/sensor.waze_travel_time.markdown [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+description: "Instructions on how to add Waze travel time to Home Assistant."
+date: 2018-01-23 00:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: waze.png
+ha_category: Transport
+ha_iot_class: "Cloud Polling"
+ha_release: 0.63
```
Comment (frenck) [inline source/_components/sensor.waze_travel_time.markdown]: `0.63` has been release some time ago. Please set it to the next version.

**Comment by frenck** [comment]
Rebased the PR in order to fix the build

**Review by frenck — DISMISSED** [review]

**Comment by frenck** [comment]
Thanks, @Myrddyn1! 🎖 

Can be merged as soon as the parent PR gets merged.

**Review by fabaff — APPROVED** [review]

**Pull request merged**